examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -165,13 +174,14 @@
 
 # This object uses method='kmeans' for KMeans clustering with 10mm smoothing
 # and standardization ON
-kmeans = Parcellations(method='kmeans', n_parcels=50,
+start = time.time()
+kmeans = Parcellations(method='kmeans', n_parcels=2000,
```

Comment:
```suggestion
kmeans = Parcellations(method='kmeans', n_parcels=50,
```
Can you switch this to 50 please. 2000 is not the good regime for kmeans.